utilities/audio_utils.py

```
@@ -4,113 +4,90 @@
+spk_clone_model_device = "cuda" #"cpu"
+
+def make_speaker_embedding(wav: torch.Tensor, sr: int) -> torch.Tensor:
+    global spk_clone_model
+    if spk_clone_model is None:
+        spk_clone_model = SpeakerEmbeddingLDA(device=torch.device(spk_clone_model_device))
+    if wav.device != spk_clone_model.device:
+        wav = wav.to(spk_clone_model.device)
+    _, spk_embedding = spk_clone_model(wav, sr)
+    return spk_embedding.unsqueeze(0).bfloat16()
+
+async def process_speaker_audio(speaker_audio_path: str, uuid: int, model, device: torch.device, enable_disk_cache=True) -> torch.Tensor:
```

Ruff (0.12.2)

Unused function argument: `model`

(ARG001)